app/Http/Controllers/admin/AttributeValuesController.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace App\Http\Controllers\admin;
+
+use App\Http\Controllers\Controller;
+use App\Models\AttributeTypes;
+use App\Models\AttributeValues;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+
+class AttributeValuesController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $aryAttributeValues = AttributeValues::with('attributes')->paginate(5);
+        $aryAttributeTypes = AttributeTypes::all();
+        return view('admin.attributes.value.list', compact('aryAttributeTypes', 'aryAttributeValues'));
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $validator = Validator::make(
+            $request->all(),
+            [
+                'name' => 'max:255|min:3'
+            ],
+            [
+                'max' => 'Name is too long!',
```

Comment:
show message more clearer and specific condition. Ex: Name is too long ( max: 255 character)